tests/pom.xml

```diff
@@ -56,6 +56,8 @@ Test cases are run against Docker images of Hono server + (Apache Qpid Dispatch
     <http.java.options></http.java.options>
     <mqtt.java.options></mqtt.java.options>
     <deviceregistry.image>hono-service-device-registry</deviceregistry.image>
+    <!-- logging profile: either prod, dev or trace -->
+    <logging.profile>prod</logging.profile>
```

Comment:
this enhancement doesn't seem related to the original scope of this PR. Would you mind putting it into a separate commit (no need for a PR FMPOV)?

Reply:
Yes, done.